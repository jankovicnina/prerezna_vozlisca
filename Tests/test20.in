982 1407
800 167 -1.05
832 460 6.4
199 897 8.76
637 93 -0.51
743 660 8.53
918 597 5.29
523 493 4.67
909 961 7.44
890 60 7.96
128 28 5.32
297 694 5.33
121 901 -0.76
9 569 7.65
934 112 9.41
375 523 4.49
249 355 8.29
264 328 6.66
401 767 8.67
906 185 7.66
496 283 1.88
883 316 7.74
192 244 0.27
546 202 9.07
922 925 0.15
336 398 -0.77
877 549 7.8
197 21 8.61
381 129 6.54
289 190 0.17
228 361 7.87
235 6 8.36
421 476 5.83
549 735 6.8
767 923 0.7
476 89 -1.49
324 73 8.96
52 317 3.99
689 246 -0.82
287 646 -0.26
362 743 1.2
325 364 8.34
208 59 2.76
7 556 -1.12
321 715 3.69
232 681 -0.65
539 616 3.65
54 466 9.14
177 179 3.23
54 140 2.11
408 537 9.17
312 575 1.95
463 75 8.27
867 434 7.87
470 228 -0.56
28 551 -0.21
477 512 2.35
140 918 3.53
517 871 8.73
149 156 2.44
958 413 -0.28
148 889 4.62
622 109 -0.7
1 836 5.08
963 867 8.45
274 389 8.13
431 460 3.96
97 221 9.79
33 326 3.69
697 468 2.17
693 151 6.62
804 785 6.36
802 497 1.72
597 363 9.56
454 880 8.52
73 54 3.7
106 8 4.01
911 29 9.57
531 635 3.24
143 100 9.65
910 839 0.2
16 489 6.72
812 166 3.19
102 760 1.94
529 238 6.75
651 27 -0.75
692 275 3.3
272 703 8.85
486 369 9.86
766 342 6.18
893 945 9.3
279 503 7.11
48 367 -1.49
933 13 4.33
198 533 4.0
173 170 4.13
468 114 -1.36
62 22 2.09
352 588 3.56
716 730 -0.14
47 939 2.25
593 584 8.49
361 976 2.79
603 708 0.22
775 904 7.83
138 967 2.69
525 690 8.84
250 544 2.39
186 704 5.72
696 626 -0.68
397 807 1.28
369 318 -1.3
132 604 3.87
784 538 0.78
306 959 6.97
276 273 -1.29
283 121 -1.98
239 333 8.41
564 528 2.22
309 778 0.16
850 682 4.64
812 719 6.1
518 307 6.53
788 720 -0.83
357 335 -0.27
674 295 0.18
534 670 9.07
78 541 6.12
460 728 2.53
290 630 0.71
453 74 4.3
819 452 7.75
403 492 6.08
295 684 -0.87
440 766 8.53
691 810 8.77
93 820 0.33
385 484 1.9
467 86 -1.57
199 93 3.12
34 265 4.16
12 88 1.26
585 403 7.99
23 257 6.36
68 200 8.69
40 360 -1.76
487 373 7.45
524 453 8.48
128 23 5.74
728 863 5.27
798 751 3.26
311 482 1.8
122 838 7.64
480 508 2.54
60 59 1.72
862 876 3.43
660 286 3.09
497 621 -1.96
535 958 9.28
542 832 8.3
720 916 1.31
872 415 3.1
648 731 0.42
634 423 8.54
45 596 2.6
14 292 1.0
24 742 9.53
39 298 4.92
513 496 9.84
914 696 -1.16
107 806 3.43
75 262 0.32
309 454 3.84
764 95 1.94
464 726 9.09
221 189 4.45
297 187 0.03
645 104 8.6
808 659 1.71
502 821 1.88
913 180 -1.83
950 377 9.68
160 526 4.18
690 124 6.34
437 271 8.21
875 803 -0.81
809 168 -0.3
271 139 3.06
432 397 2.15
592 974 7.92
931 749 5.93
953 706 -0.49
201 946 3.28
131 422 2.55
600 718 7.27
269 370 6.17
127 119 2.16
834 216 9.63
897 128 9.63
83 426 0.76
658 119 2.54
168 574 -0.4
494 975 -0.53
797 559 3.38
206 650 0.03
670 692 1.69
900 609 1.27
927 500 8.2
642 220 4.63
671 872 3.52
792 7 1.11
920 895 8.26
778 884 5.77
12 708 5.4
403 696 6.44
444 214 4.26
67 193 1.91
521 780 5.93
759 44 -0.34
661 445 9.87
837 610 6.24
978 952 4.3
674 455 8.91
478 268 8.35
761 148 0.97
109 927 9.51
670 30 6.85
36 486 9.31
835 546 8.93
30 61 4.64
781 980 8.21
495 608 2.0
908 46 -0.2
446 68 4.29
536 210 4.91
71 280 6.88
471 256 8.41
647 229 3.01
47 232 -0.1
952 161 4.11
488 418 -1.2
736 35 1.01
436 260 8.48
125 300 4.6
78 579 8.81
378 670 0.45
189 940 -1.13
537 507 6.63
827 431 0.36
420 116 -1.69
919 495 2.65
808 258 1.51
305 479 -0.45
834 711 4.56
624 312 6.57
599 687 -0.56
205 828 3.9
945 571 9.37
962 874 6.33
118 842 1.04
738 320 5.39
894 760 9.52
725 835 3.22
815 822 3.51
735 965 4.81
156 97 4.46
865 658 -0.97
581 981 6.62
705 652 6.51
400 83 3.09
766 275 0.53
773 643 4.47
936 723 -1.6
602 40 6.94
185 779 3.11
277 0 0.4
559 839 1.51
316 269 -0.66
176 230 -0.05
687 83 4.37
358 595 3.11
63 150 9.4
960 454 8.22
69 771 5.37
892 251 3.24
180 577 1.2
155 715 3.4
520 619 3.21
722 594 0.03
627 598 3.32
8 816 4.9
780 383 5.6
344 358 2.69
909 117 4.04
634 56 8.29
686 748 2.74
675 561 8.81
420 922 -1.27
915 281 2.24
858 860 -0.57
477 155 7.58
423 546 0.18
362 649 3.93
874 824 -1.79
593 467 8.45
187 331 6.68
370 741 3.7
135 337 9.97
684 702 6.38
554 599 5.37
530 88 0.99
577 11 3.72
188 970 8.49
859 936 -0.43
257 30 8.93
852 689 5.3
566 717 3.43
97 846 9.48
843 488 7.42
292 199 7.72
567 960 5.73
864 664 3.38
960 862 7.68
830 515 5.76
784 491 8.13
680 134 1.57
459 796 3.52
456 902 6.85
527 622 4.65
146 755 9.59
813 888 4.6
551 227 6.84
472 392 3.22
45 527 9.53
926 187 3.5
377 784 8.02
91 168 -1.14
115 297 7.4
836 733 -0.98
153 91 0.43
465 809 1.26
265 30 9.33
667 645 5.19
419 321 4.55
368 37 2.42
923 70 5.21
959 308 -1.13
500 875 -0.38
507 583 8.12
60 858 9.24
579 909 3.36
834 490 0.19
368 605 6.93
958 12 4.22
481 614 8.71
553 518 5.93
227 945 4.2
848 878 7.5
72 779 4.0
353 511 2.48
315 625 5.0
289 592 0.15
681 176 6.15
472 154 7.34
706 567 6.94
855 34 9.12
797 408 4.51
842 611 1.45
152 788 -1.45
902 833 6.09
437 136 8.44
10 527 5.8
708 512 5.53
447 699 5.89
117 36 -0.01
19 287 3.53
776 977 -0.27
247 504 9.63
213 631 0.75
607 424 7.8
43 843 0.53
172 847 -1.86
433 937 9.05
969 254 7.37
875 273 2.43
845 755 5.05
943 922 9.79
90 747 4.77
155 37 1.29
830 804 6.05
854 395 3.26
510 231 0.32
254 212 -0.11
191 600 3.86
731 659 8.71
659 372 2.93
388 448 7.72
619 926 3.47
460 688 -0.51
129 43 4.62
138 560 2.34
728 164 3.19
63 653 3.73
650 908 -2.0
270 303 4.47
63 906 6.57
24 448 4.29
700 368 10.0
594 26 4.62
418 366 0.65
151 573 4.82
229 336 9.33
36 802 3.05
742 485 6.27
856 243 -1.99
55 662 8.7
980 450 0.33
402 680 7.42
499 598 5.56
724 187 8.65
544 165 -0.78
431 205 5.0
121 572 6.04
846 164 9.06
919 317 5.92
605 936 7.29
352 356 -0.32
411 219 6.73
208 764 1.99
608 869 7.0
470 739 7.08
596 828 1.62
110 123 4.55
825 29 4.63
938 120 8.58
635 75 1.23
829 358 9.72
488 677 7.28
421 319 9.74
831 131 0.98
459 891 8.0
712 185 1.48
727 911 3.81
518 378 8.12
499 66 6.2
519 621 0.03
852 781 9.31
235 83 8.95
683 214 -0.54
262 959 3.62
84 815 0.8
263 666 9.72
330 890 -0.54
703 204 1.69
576 910 0.97
916 416 6.03
814 552 3.44
453 961 8.25
293 490 5.11
237 885 -1.2
517 856 3.99
387 732 0.46
444 657 0.05
200 396 3.35
818 163 0.92
825 232 5.82
277 451 1.82
163 348 0.57
511 18 6.25
32 542 6.34
158 182 5.36
567 119 6.14
64 795 5.95
827 972 9.06
219 755 9.64
386 157 8.58
194 333 9.25
133 547 -1.0
492 576 -1.12
334 386 3.34
92 821 7.91
347 462 3.51
74 790 4.78
692 639 1.07
608 172 8.41
154 691 9.75
569 10 4.87
959 326 -0.45
574 897 3.59
299 589 3.19
956 948 -0.33
29 887 2.66
724 911 6.08
465 273 -0.55
582 426 3.38
139 147 2.47
975 400 6.52
462 305 4.32
15 917 1.55
462 224 1.44
582 24 -0.58
373 466 8.79
434 122 8.56
712 411 2.08
398 519 4.54
533 17 2.25
327 818 6.48
694 607 9.99
196 472 6.75
3 420 6.0
26 298 4.38
116 638 2.45
884 783 1.46
902 735 7.41
887 81 5.8
390 840 4.91
119 519 -1.66
649 716 6.48
243 353 6.01
452 772 7.29
321 147 2.77
957 145 3.33
897 849 2.57
112 314 -1.56
552 776 5.42
706 40 5.52
41 548 5.04
745 873 1.87
893 391 3.39
55 719 3.56
49 301 4.4
644 191 -1.82
878 236 3.71
35 382 5.96
232 844 4.33
720 460 -1.28
970 208 5.99
882 905 8.39
431 252 0.84
393 325 -1.85
792 836 8.6
845 789 0.21
167 899 3.15
332 44 4.61
741 464 4.81
493 394 6.0
236 756 -1.91
596 390 7.4
440 588 6.46
267 259 1.78
388 126 4.72
672 324 4.99
733 535 -0.92
82 103 5.74
145 933 7.9
280 599 -1.11
300 915 0.79
676 278 -0.65
365 458 2.7
928 529 8.44
307 416 7.96
56 640 7.72
979 215 6.87
275 137 4.29
870 670 0.55
443 19 0.63
685 942 2.48
252 626 8.19
308 67 6.71
504 364 -1.64
655 53 1.28
179 28 1.79
547 655 2.95
325 739 3.46
882 629 0.4
833 556 6.86
875 275 6.44
816 714 0.68
322 20 7.19
243 683 2.94
296 24 0.47
501 57 8.14
387 199 8.8
782 327 1.05
869 264 -1.71
588 413 1.49
844 125 3.17
880 824 -1.26
820 622 5.38
474 851 5.63
590 101 4.23
399 581 2.82
275 671 0.22
798 457 8.78
579 823 0.32
448 695 0.15
322 19 9.13
846 467 5.2
79 952 5.05
166 213 -1.79
564 65 5.04
971 55 6.17
563 338 8.77
86 868 5.2
230 920 8.58
578 791 5.4
22 612 3.71
220 68 3.03
748 454 1.08
593 801 -0.67
942 887 6.78
72 494 6.14
868 50 3.83
76 443 0.64
876 7 5.51
58 478 3.15
382 730 -0.22
823 620 7.61
609 650 8.31
980 910 3.42
395 474 9.32
458 525 3.57
695 520 1.49
787 88 9.3
795 744 3.39
485 866 4.23
287 78 1.54
184 724 4.76
409 113 -0.41
852 737 2.0
569 545 4.65
866 510 1.04
614 656 7.5
301 146 9.76
19 446 0.18
72 420 7.35
135 292 -0.54
811 590 8.46
379 545 -1.11
713 15 5.24
691 401 9.05
628 777 5.98
751 277 8.48
484 201 9.25
359 912 5.49
312 317 8.38
935 171 8.82
696 568 3.12
898 615 5.74
715 498 -0.08
839 665 3.02
196 123 9.15
291 399 1.17
339 149 9.98
757 655 7.43
242 712 2.77
230 447 6.2
886 881 4.34
657 505 -1.34
516 279 -0.28
979 177 0.88
19 285 8.23
824 574 5.28
850 41 -1.91
63 668 6.23
233 822 -1.36
113 183 5.33
204 399 4.38
540 746 5.97
585 713 7.56
156 450 8.79
622 774 1.46
314 807 6.17
286 184 -0.24
466 790 -0.91
162 963 5.56
746 319 -1.06
570 515 -1.69
329 156 -1.15
281 364 2.16
253 817 7.29
489 332 3.28
44 189 6.36
337 175 0.27
641 669 5.69
475 845 9.91
396 721 5.71
555 675 9.09
358 613 4.06
218 837 8.1
107 442 -0.14
716 72 1.58
173 631 2.96
676 445 0.38
665 967 0.15
889 767 1.22
210 613 3.69
317 909 6.39
147 37 6.0
939 633 8.0
3 135 3.05
559 159 3.98
261 935 6.98
127 903 6.36
753 757 4.51
127 869 7.06
557 882 4.56
558 267 2.25
721 536 9.67
509 520 2.48
102 632 -1.51
771 825 2.55
699 438 2.39
471 134 8.72
976 142 1.74
638 683 -0.53
702 848 -1.38
171 418 4.01
860 290 8.76
89 738 4.0
282 416 4.18
587 347 1.99
887 168 2.46
651 320 -0.99
291 274 6.2
875 484 0.12
623 875 6.35
625 247 -1.2
762 201 -0.4
744 593 8.75
203 796 7.8
753 911 8.77
671 974 6.0
103 25 0.5
445 365 3.59
625 304 9.64
164 955 9.46
435 102 6.57
204 236 8.22
612 855 -1.59
562 143 3.42
1 85 2.12
792 672 5.04
810 445 8.67
820 289 8.81
294 429 9.71
930 437 9.71
885 807 7.72
284 501 -0.12
290 671 8.04
148 2 5.74
174 71 9.46
616 267 4.72
281 509 6.35
333 968 -0.46
87 705 9.71
778 187 4.24
805 346 7.62
691 110 -0.11
209 929 -1.75
610 871 8.22
251 492 2.43
216 459 1.03
857 334 8.64
314 393 7.41
46 214 9.06
776 641 1.87
224 176 -0.78
810 169 10.0
906 494 6.52
355 253 -1.23
978 501 1.32
280 419 8.65
657 823 3.36
506 966 9.58
201 668 0.6
880 530 7.52
392 144 8.83
807 471 5.35
972 697 7.26
630 793 1.53
606 636 2.59
788 36 7.88
560 915 -1.42
631 499 1.3
701 146 8.67
311 390 8.61
24 842 1.48
35 620 2.24
21 388 1.37
925 679 8.44
360 249 8.38
154 191 1.73
632 568 9.69
739 184 0.66
774 105 6.43
822 736 -1.12
179 771 3.89
91 14 -1.44
245 758 4.21
396 392 -1.06
898 913 8.78
619 778 9.36
244 255 0.63
366 539 -0.96
85 469 1.57
821 654 0.96
410 446 2.39
37 181 8.0
32 294 1.76
604 841 4.18
100 351 0.47
234 433 4.68
114 707 -0.91
429 505 4.06
541 849 -1.57
295 932 5.63
428 603 0.17
222 481 5.57
586 90 6.78
96 894 5.93
182 479 -0.84
263 102 8.78
656 636 8.06
765 663 -1.29
169 405 1.76
745 177 1.84
412 192 -1.02
190 846 2.63
791 944 0.77
723 892 6.75
823 310 7.39
719 427 0.68
726 293 4.93
926 414 6.64
313 52 2.75
723 272 1.17
268 78 4.9
286 306 7.35
736 713 -1.26
142 411 6.43
954 3 3.26
617 561 2.0
859 682 4.61
13 197 3.22
817 589 4.23
517 483 -1.22
246 668 2.2
688 137 3.45
366 527 6.95
350 494 5.56
654 473 7.53
332 414 5.88
294 638 7.95
755 412 7.6
305 487 9.45
357 252 7.57
636 732 2.84
308 966 9.5
733 910 8.18
28 380 0.74
949 514 9.99
789 98 5.32
300 107 4.54
343 428 3.51
941 111 2.86
212 404 5.5
746 444 -0.01
225 676 -0.74
101 491 4.95
903 245 3.19
331 79 9.95
159 432 0.67
180 615 7.58
318 296 1.54
835 354 7.34
852 880 7.84
95 449 3.46
575 763 3.1
615 725 6.59
725 288 0.8
806 142 8.43
948 778 -1.01
401 302 0.91
424 954 3.5
215 582 9.37
795 236 7.28
980 814 1.78
397 812 1.86
672 309 2.44
490 901 8.09
714 663 3.26
424 814 -0.8
656 558 -1.5
826 857 3.69
734 166 6.27
321 1 -0.7
763 620 -1.91
617 409 8.44
287 544 -0.85
758 276 0.87
789 956 2.32
929 805 -1.56
623 566 0.8
12 341 2.94
438 569 4.57
57 436 7.3
400 110 1.51
170 584 6.41
757 340 1.65
475 723 2.7
556 688 4.29
57 0 0.82
181 141 4.37
142 32 1.33
665 516 7.11
199 158 0.95
271 969 9.24
344 957 2.07
905 673 0.05
365 269 8.53
701 919 0.23
79 617 0.6
460 513 2.65
97 647 3.69
941 580 2.23
322 135 -1.68
405 930 8.96
286 524 6.22
851 190 2.73
144 291 6.08
108 185 2.17
119 252 4.09
318 941 0.33
165 359 4.66
689 883 9.18
319 769 5.8
679 33 5.58
515 878 -0.18
374 500 9.38
240 270 6.7
707 335 -1.83
777 753 9.47
18 651 9.26
11 938 7.14
302 562 5.99
611 924 1.07
571 444 7.74
889 486 8.15
532 299 7.31
879 800 -0.95
752 648 5.3
549 700 -0.86
512 879 5.4
12 210 8.08
257 643 3.99
718 309 5.04
981 765 -1.17
372 678 -0.11
405 289 5.46
455 826 8.98
124 773 0.66
584 865 5.1
479 449 7.81
828 330 5.19
192 800 -0.5
130 629 -1.58
480 733 6.68
276 740 3.52
629 715 0.13
785 781 -0.97
921 339 3.64
83 483 1.37
653 506 6.96
709 207 -0.3
134 947 0.11
978 792 4.83
729 313 6.33
961 896 0.91
573 233 4.9
772 487 5.43
678 153 8.58
551 951 1.64
927 746 9.57
448 426 -0.68
425 408 7.93
440 867 -0.49
820 42 0.17
38 348 7.88
252 242 -1.56
278 517 7.35
60 551 4.54
598 65 8.1
826 745 -0.37
267 236 4.77
955 19 4.46
22 428 3.88
801 49 9.57
323 808 4.7
193 362 9.66
95 132 5.18
260 306 -1.65
310 261 8.77
356 914 2.41
81 578 0.86
668 829 8.87
37 127 8.05
940 950 6.82
419 593 8.02
537 709 4.54
207 786 -0.56
299 873 5.07
462 974 7.64
740 628 -0.59
580 898 2.19
182 344 -0.99
572 108 0.28
241 227 5.63
944 964 -0.62
770 830 2.08
338 949 7.89
8 768 9.66
748 96 1.76
217 956 9.74
272 913 9.82
479 532 0.99
618 31 -1.62
605 376 3.54
530 196 -2.0
901 813 8.56
266 522 2.8
813 945 3.67
177 963 1.85
141 761 7.1
429 540 0.4
9 139 -0.34
231 932 -1.24
260 783 7.42
373 948 3.42
473 565 3.2
90 77 8.0
876 567 7.75
589 13 5.1
66 931 7.31
863 442 -0.5
77 943 1.3
633 384 9.28
342 661 6.23
940 792 3.28
333 539 3.91
639 793 5.8
397 378 3.89
681 576 1.99
965 475 2.14
646 106 -0.16
593 977 2.61
259 892 8.7
134 78 4.96
44 906 -0.83
441 928 5.11
466 861 6.93
788 613 2.93
900 764 -0.99
640 29 6.53
917 250 2.54
443 782 3.71
748 48 7.67
312 428 6.45
626 922 9.13
150 352 9.98
821 532 9.11
940 623 4.61
37 616 1.45
505 445 9.65
212 735 1.74
144 65 5.14
601 224 5.48
870 857 5.34
366 734 4.41
43 671 -1.3
226 543 7.52
808 630 8.0
779 422 7.84
70 834 9.6
841 51 9.63
422 287 4.8
663 856 4.61
340 889 0.96
223 903 7.78
943 276 5.48
383 406 6.29
60 14 6.4
239 96 5.7
499 184 0.11
946 76 8.61
380 674 3.44
384 701 8.0
799 814 -1.05
170 128 7.78
30 195 7.92
31 132 5.06
328 35 0.63
120 624 2.01
962 61 2.75
439 521 6.65
206 441 4.16
750 952 9.7
882 468 3.93
806 602 -0.93
134 740 -0.53
786 402 1.2
433 872 0.63
818 182 0.8
613 586 0.14
482 425 9.18
673 195 0.71
361 828 4.52
149 94 5.08
853 950 9.03
771 805 0.67
677 223 9.25
367 1 0.91
666 953 -0.72
341 307 -0.41
514 722 8.52
526 685 0.05
416 811 4.22
508 329 4.11
380 898 3.9
168 698 7.86
583 852 8.8
760 178 6.07
910 333 5.2
80 972 2.97
0 407 6.57
6 819 9.25
522 770 1.83
780 178 -1.28
94 605 3.09
282 973 7.98
621 174 -0.49
661 463 -1.16
891 282 9.22
17 239 6.64
881 0 5.56
27 615 2.41
765 554 2.1
21 571 5.53
195 439 1.78
774 39 -0.6
387 686 -0.19
335 465 4.95
943 625 -1.18
42 634 5.07
256 152 9.62
769 69 1.98
38 349 4.9
354 541 6.17
417 430 4.0
895 225 5.15
224 316 0.83
531 79 9.73
146 33 7.26
941 66 9.18
72 10 2.79
124 142 5.16
161 843 -0.41
717 63 2.9
833 92 0.59
299 768 1.54
652 542 -1.84
681 194 5.35
682 333 6.65
123 644 5.87
629 16 2.4
0 441 5.81
178 401 9.82
4 295 0.81
973 235 7.78
534 606 8.41
554 385 4.75
349 886 -1.05
244 817 3.7
602 601 2.01
2 234 0.97
328 836 5.05
298 115 8.47
99 799 5.51
264 338 5.45
649 336 5.92
25 502 1.28
347 77 9.43
790 198 2.45
947 642 2.38
794 667 9.12
157 553 8.78
10 276 1.84
140 53 9.72
565 462 9.97
967 883 9.95
9 768 0.62
974 178 3.05
491 466 -0.62
391 117 3.51
183 794 4.06
698 263 -1.47
550 858 8.25
427 247 -0.63
607 133 -1.11
62 332 9.0
861 570 1.25
776 329 5.86
873 311 7.77
51 218 5.67
860 283 8.08
279 947 9.79
457 587 4.47
803 467 8.04
138 297 6.29
591 461 5.24
296 9 1.93
548 680 -1.12
809 742 1.48
255 186 2.96
275 181 5.67
320 62 1.74
756 381 4.04
414 375 3.43
849 58 0.04
912 831 7.1
413 797 6.87
704 241 6.96
937 38 9.59
635 853 3.91
351 854 -1.87
561 323 -0.26
922 844 7.01
513 635 4.9
136 727 5.99
914 827 -1.98
796 84 -1.38
679 503 5.16
111 457 -1.31
596 118 2.35
415 647 0.75
211 99 1.58
729 838 8.55
493 728 -0.63
50 893 7.83
595 564 -0.69
363 798 -1.88
643 627 -0.76
202 45 7.11
304 724 9.68
53 392 0.96
469 173 3.55
896 211 9.74
503 752 -0.96
5 470 -0.83
760 934 4.55
620 59 -1.38
303 754 2.85
600 156 5.96
848 477 5.65
285 203 -1.11
782 458 -0.91
368 387 1.1
389 64 -0.8
545 563 4.34
304 5 3.6
350 709 9.61
593 257 7.3
152 169 8.07
346 215 8.69
483 266 6.12
964 40 8.06
61 550 2.81
916 188 1.37
498 850 4.9
711 47 9.92
793 450 6.76
288 775 -1.03
20 206 -1.29
927 930 9.3
245 549 3.44
332 669 8.84
737 288 6.17
194 920 9.0
942 591 7.05
926 374 0.22
502 480 6.94
815 693 9.71
747 222 4.4
192 39 6.08
589 162 1.05
754 105 -1.1
309 284 4.38
683 421 0.89
795 306 -0.33
407 585 5.16
968 271 0.29
175 681 4.08
205 94 4.31
616 279 3.93
899 161 0.34
406 410 7.76
749 711 -1.05
538 80 8.7
245 17 3.57
155 1 0.68
137 859 5.6
80 825 3.89
730 343 -1.62
450 877 0.46
394 979 9.66
981 68 3.0
115 759 9.35
376 534 8.27
238 967 2.06
449 357 5.01
630 907 9.59
364 516 9.19
907 126 8.08
516 138 6.07
695 951 1.92
836 167 2.9
72 296 -0.87
524 462 -0.71
665 111 0.73
65 962 0.77
484 87 9.11
39 870 1.58
326 836 3.86
853 59 -0.76
967 197 6.22
664 539 9.74
461 864 -0.51
88 376 7.44
714 254 9.37
710 379 2.93
732 978 -0.69
904 322 0.33
568 226 3.76
783 424 3.69
7 304 6.95
669 653 5.77
922 112 7.58
731 73 -1.01
306 248 -1.93
273 599 4.51
604 428 -0.97
363 525 3.96
956 762 1.97
838 641 6.85
899 130 0.84
528 750 0.33
476 10 5.88
186 877 2.09
543 240 8.73
626 237 4.23
951 787 6.8
464 555 9.15
911 371 2.95
345 217 1.97
321 637 -1.17
957 276 3.88
104 315 0.01
348 82 8.06
426 900 3.88
845 570 6.36
404 971 7.31
446 531 7.51
371 423 2.7
126 4 3.92
214 456 8.96
257 109 -1.26
388 381 8.86
398 729 -1.25
98 209 4.63
758 40 5.72
451 345 5.88
662 710 -1.85
783 550 2.35
519 921 3.97
977 350 -0.39
442 394 1.47
258 639 -1.27
548 356 6.27
918 297 7.56
924 759 9.47
932 440 3.2
768 390 -1.32
576 417 5.71
248 700 4.2
888 435 2.44
105 557 6.26
241 555 1.69
847 20 9.99
840 618 1.85
966 284 -1.13
644 230 0.89
466 698 0.94
871 767 2.21
430 743 -0.03
779 689 9.7
184 19 9.89
782 149 6.44
59 160 6.34
