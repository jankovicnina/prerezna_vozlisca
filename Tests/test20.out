65.33266382032811
